pkgs/build-support/rust/build-rust-crate/configure-crate.nix

```diff
@@ -147,6 +154,11 @@ in ''
   export CARGO_PKG_VERSION_PATCH=${lib.elemAt version 2}
   export CARGO_PKG_VERSION_PRE="${versionPre}"
   export CARGO_PKG_HOMEPAGE="${crateHomepage}"
+  export CARGO_PKG_LICENSE="${crateLicense}"
+  export CARGO_PKG_LICENSE_FILE="${crateLicenseFile}"
+  export CARGO_PKG_README="${crateReadme}"
+  export CARGO_PKG_REPOSITORY="${crateRepository}"
+  export CARGO_PKG_RUST_VERSION="${crateRustVersion}"
```

Comment:
This unconditionally overrides the environment variable with an empty string in the case that `crate.rust-version` is unset. This is not set anywhere in nixpkgs or in crate2nix. Therefore this just clobbers the environment variable, breaking code that used to work before this change.

Reply:
Even if I weren't 9 months late on this, I think this is crate2nix's problem, not buildRustCrate's problem.

Users of crate2nix can override this, assuming they know their MSRV and have it in a `msrv` variable, by doing

```nix
buildRustCrateForPkgs = pkgs: crate: pkgs.buildRustCrate (crate // { rust-version = msrv; })
```

which is a little ugly but arguably cleaner than the previous workaround (using `crate.override.preUnpack` to set the environment variables directly).